starter-code/french-onion-soup-starter-code/index.html

```diff
@@ -2,42 +2,67 @@
 <html lang="en">
 <head>
   <title>French Onion Soup Recipe</title>
-  <!-- Google fonts gives us access to the Open Sans Font -->
-  <!-- Google fonts is also awesome! Make sure to check it out -->
-  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Raleway" rel="stylesheet">
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-  French Onion Soup
+  
+  <section class = "title">
+    <h2>French Onion Soup</h2>
+  </section> 
 
-  We have been trying French onion soup in restaurants for years and my family
+  <br>
+
+  <p>We have been trying French onion soup in restaurants for years and my family
 and friends agree none can compare to my recipe for taste and simplicity of
-preparation.
-
-Ingredients
-1/2 cup unsalted butter
-2 tablespoons olive oil
-4 cups sliced onions
-4 (10.5 ounce) cans beef broth
-2 tablespoons dry sherry (optional)
-1 teaspoon dried thyme
-salt and pepper to taste
-4 slices French bread
-4 slices provolone cheese
-2 slices Swiss cheese, diced
-1/4 cup grated Parmesan cheese
-
-Directions
-Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
-continually stir until tender and translucent. Do not brown the onions.
-
-Add beef broth, sherry and thyme. Season with salt and pepper, and simmer for 30
-minutes.
-
-Heat the oven broiler.
-
-Ladle soup into oven safe serving bowls and place one slice of bread on top of each
+preparation.</p>
+
+<img class ="info" src="./images/recipe-info.png" alt="Recipe Info">
+
+<hr>
+
+<div>
+    <h3>Ingredients</h3>
+    <ul class="dashed">
+      <li> 2 tablespoons olive oil</li>
+      <li> 4 cups sliced onions</li>
+      <li> 4 (10.5 ounce) cans beef broth</li>
+      <li> 2 tablespoons dry sherry (optional)</li>
+      <li> 1 teaspoon dried thyme</li>
+      <li> salt and pepper to taste</li>
+      <li> 4 slices French bread</li>
+      <li> 4 slices provolone cheese</li>
+      <li> 2 slices Swiss cheese, diced</li>
+      <li> 1/4 cup grated Parmesan cheese</li>
+      <li> 1/2 cup unsalted butter</li>
+    </ul>
+
+    <img class ="info" src="./images/cooking-info.png" alt="Cooking Info">
+
+</div>
+
+<hr>
+
+<div>
+<h3>Directions</h3>
+
+<ol>
+<li>Melt butter with olive oil in an 8 quart stock pot on medium heat. Add onions and
```

Comment:
this list should have had the number then ) instead of a period(.). One way to do this is to just do list-style: none then add 1) 2) etc in front of the text of each list. or you can use the pro way and in your css folder do this
`
ol {
counter-reset: custom;
list-style-type: none;
line-height: 1.6;
margin: 0 auto;
}

ol li:before {
content: counter(custom)') ';
counter-increment: custom;
}
`